core/build.sbt

```diff
@@ -459,7 +458,8 @@ lazy val operatorActions =
       libraryDependencies ++= Vector(
             AkkaSlf4j,
             Ficus,
-            Logback,
+            LogbackClassic,
```

Comment:
I guess we should not do tis in operatorActions (we should remove it, add dependency to operator)